ntbs-service/Services/NotificationService.cs

```diff
@@ -467,6 +475,13 @@ public async Task DenotifyNotificationAsync(int notificationId, DenotificationDe
             notification.NotificationStatus = NotificationStatus.Denotified;
 
             await UpdateDatabaseAsync(NotificationAuditType.Denotified);
+            
+            Log.Debug($"{notificationId} denotifyied, removing lab result matches");
```

Comment:
Typo in 'denotified' (and similar in the project level dictionary).

Reply:
well that's awkward :P thanks for spotting that!